app/decorators/service_decorator.rb

```diff
@@ -68,7 +68,7 @@ def new_application_data
   end
 
   def backends_table_data
-    BackendApiDecorator.decorate_collection(backend_apis)
+    BackendApiDecorator.decorate_collection(backend_apis.sort_by(&:name))
```

Comment:
Is backend_apis an `Array` or an `ActiveRecord::Collection` ?

if the latter, then we should use 
```suggestion
    BackendApiDecorator.decorate_collection(backend_apis.desc(name: :asc))
```

Reply:
@hallelujah Do you mean
```suggestion
    BackendApiDecorator.decorate_collection(backend_apis.order(name: :asc))
```

Reply:
It is a `BackendApi::ActiveRecord_Associations_CollectionProxy` and it doesn't have that sort method.

Reply:
I do not get why `order` does not work

```ruby
[2] pry(main)> service.backend_apis.order(name: :asc)                                                                                                                                               
  BackendApi Load (1.4ms)  SELECT `backend_apis`.* FROM `backend_apis` INNER JOIN `backend_api_configs` ON `backend_apis`.`id` = `backend_api_configs`.`backend_api_id` WHERE `backend_api_configs`.`service_id` = 2 ORDER BY `backend_api_configs`.`id` ASC, `backend_apis`.`name` ASC
=> [#<BackendApi:0x000000000cde9958
  id: 2,
  name: "API Backend",
  system_name: "api",
  description: "Backend of API",
  private_endpoint: "https://echo-api.3scale.net:443",
  account_id: 2,
  created_at: Tue, 20 Apr 2021 12:41:24 UTC +00:00,
  updated_at: Tue, 20 Apr 2021 12:41:24 UTC +00:00,
  tenant_id: nil,
  state: "published">]
```

Reply:
Done